sdk/identity/identity/test/manual-integration/AzureVM/index.ts

```diff
@@ -0,0 +1,21 @@
+import { SecretClient } from "@azure/keyvault-secrets";
+import { DefaultAzureCredential } from "@azure/identity";
+
+async function main(): Promise<void> {
+  // DefaultAzureCredential expects the following three environment variables:
```

Comment:
When expected to use managed identity, it expects no environment variables.

Reply:
Copy/paste error. I'll fix